.github/workflows/package.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
         name: ckb_${{env.GIT_TAG_NAME }}_${{env.REL_PKG }}.asc
         path: ckb_${{env.GIT_TAG_NAME }}_${{env.REL_PKG }}.asc
     env:
-      BUILDER_IMAGE: nervos/ckb-docker-builder:bionic-rust-1.71.1
+      BUILDER_IMAGE: nervos/ckb-docker-builder:bionic-rust-1.71.1-openssl-3.1.3
```

Comment:
`nervos/ckb-docker-builder:bionic-rust-1.71.1-openssl-3.1.3` doesn't exist in dockerhub.
In dockerhub, we have `[bionic-rust-1.71.0-openssl-3.1.3](https://hub.docker.com/layers/nervos/ckb-docker-builder/bionic-rust-1.71.0-openssl-3.1.3/images/sha256-006f771d3493ad1508fdea1c49c0522129bc5d1e264a8149e20bedca20911654?context=explore)` and [centos-7-rust-1.71.0-openssl-3.1.3](https://hub.docker.com/layers/nervos/ckb-docker-builder/centos-7-rust-1.71.0-openssl-3.1.3/images/sha256-5c0542482215b334c03cc261a1eacb4b6bd28dff70bee0f322b60b52a3452670?context=explore)